src/Pact/Analyze/Check.hs

```diff
@@ -263,10 +264,11 @@ verifyFunctionInvariants' funInfo tables pactArgs body = runExceptT $ do
       -- assertion stack.
       ExceptT $ fmap Right $
         SBV.query $
-          for2 resultsTable $ \(Located info (AnalysisResult prop ksProvs)) -> do
+          -- TODO(joel): warn for queries that could fail!
```

Comment:
@bts I'd like to chat with you about how to do this.

Reply:
sounds good